dgm-builder/src/main/java/com/cloudbees/groovy/cps/tool/Driver.java

```diff
@@ -1,10 +1,6 @@
 package com.cloudbees.groovy.cps.tool;
 
 import com.sun.codemodel.writer.FileCodeWriter;
-import com.sun.tools.javac.api.JavacTool;
-import com.sun.tools.javac.file.JavacFileManager;
-import com.sun.tools.javac.file.RelativePath.RelativeDirectory;
-import com.sun.tools.javac.file.ZipArchive;
```

Comment:
This class no longer exists as far as I can tell.

Reply:
As of [JDK-8059976](https://bugs.openjdk.org/browse/JDK-8059976), as I discovered in https://github.com/jenkins-infra/backend-extension-indexer/pull/41.